_integration-schemas/adroll/v1/segments.md

```diff
@@ -0,0 +1,82 @@
+---
+tap: "adroll"
+version: "1"
+key: ""
+
+name: "segments"
+doc-link: "https://developers.nextroll.com/docs/crud-api/reference.html#get--api-v1-segment-get"
+singer-schema: "https://github.com/singer-io/tap-adroll/blob/master/tap_adroll/schemas/segments.json"
+description: |
+  The `{{ table.name }}` table contains information about segments, or the lists of users that visit your {{ integration.display_name }} site.
+
+replication-method: "Full Table"
+
+api-method:
+    name: ""
+    doc-link: "https://developers.nextroll.com/docs/crud-api/reference.html#get--api-v1-segment-get"
```

Comment:
Similar issue to the endpoint in `ad_groups`, this isn't the endpoint the tap uses to get this data.